src/lib.rs

```diff
@@ -313,19 +267,20 @@ impl GrowableBloom {
     /// bloom.insert(&vec![1, 2, 3]);
```

Comment:
Could we update the doc comment to match the new API? And definitely clarify in the doc comment that if you get true, your item definitely wasn't in the filter (it's easy to confuse this)